science-dev/Core/Inc/servo.h

```diff
@@ -1,28 +1,31 @@
 #ifndef SERVO_H_
 #define SERVO_H_
 
+#include <stdlib.h>
+#include "stm32g0xx_hal.h"
+
 // SG90 Servo Motor
```

Comment:
Add some notes here on how to set once frequency (e.g.

Duty cycle should be 20 ms long. Should be high for between 1-2 ms.